### PR TITLE
Adds species defines for all species

### PR DESCRIPTION
**Pull request by mikomyazaki**
Replaces all species strings with defines to help avoid mistakes with writing out species names manually.

All future PRs should use these defines rather than strings. Or risk having angry red text all over their PR from me. >:(

Probably fixes some very small bugs with sprite sheets, whitelists and more.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!review

File: code/__defines/mobs.dm [hunk 1]
```diff
@@ -322,4 +322,82 @@
...
+#define SPECIES_AUTAKH_UNATHI "Aut'akh Unathi"
+// Vaurca
+#define SPECIES_VAURCA_WORKER "Vaurca Worker"
+#define SPECIES_VAURCA_WARRIOR "Vaurca Warrior"
+#define SPECIES_VAURCA_BREEDER "Vaurca Breeder"
+#define SPECIES_VAURCA_WARFORM "Vaurca Warform"
+// Tajara
+#define SPECIES_TAJARA "Tajara"
+#define SPECIES_ZHAN_KHAZAN_TAJARA "Zhan-Khazan Tajara"
+#define SPECIES_MSAI_TJARA "M'sai Tajara"
```
Comment (Alberyk) [inline code/__defines/mobs.dm]: Should be SPECIES_MSAI_TAJARA, this typo also repeats itself in all instances.

**Comment by Alberyk** [comment]
Other issue: you are mistaking species with body types when it comes down to stuff like clothing restriction. Machine is not a valid species, but it is a valid body type.

File: code/__defines/mobs.dm (hunk 1 above)
Comment (Alberyk) [inline code/__defines/mobs.dm]: Should be SPECIES_MSAI_TAJARA, the typo also repeats itself in all stances.

File: code/game/machinery/suit_storage_unit.dm [hunk 2]
```diff
@@ -609,7 +609,7 @@
 	//Departments that the cycler can paint suits to look like.
 	var/list/departments = list("Engineering","Mining","Medical","Security","Atmos")
 	//Species that the suits can be configured to fit.
-	var/list/species = list("Human","Skrell","Unathi","Tajara", "Vaurca", "Machine")
+	var/list/species = list(SPECIES_HUMAN,SPECIES_SKRELL,SPECIES_UNATHI,SPECIES_TAJARA, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_BREEDER, SPECIES_VAURCA_WARFORM, SPECIES_IPC)
```
Comment (Alberyk) [inline code/game/machinery/suit_storage_unit.dm]: This uses bodytypes, not species names. Also, vaurca warform and breeders are not playable species and should not show up here.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 3]
```diff
@@ -28,7 +28,7 @@
 	var/gender = NEUTER
 
 	// Restrict some styles to specific species
-	var/list/species_allowed = list("Human")
+	var/list/species_allowed = list(SPECIES_HUMAN)
```
Comment (Alberyk) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: This also uses bodytypes, not species' names.

File: code/modules/clothing/spacesuits/void/void.dm [hunk 4]
```diff
@@ -15,20 +15,20 @@
 		)
 
 	//Species-specific stuff.
-	species_restricted = list("Human", "Bishop Accessory Frame", "Zeng-Hu Mobility Frame")
+	species_restricted = list(SPECIES_HUMAN, SPECIES_BISHOP_IPC, SPECIES_ZENGHU_IPC)
```
Comment (Alberyk) [inline code/modules/clothing/spacesuits/void/void.dm]: This also uses bodytypes, not species's name.

File: code/modules/clothing/spacesuits/void/void.dm [hunk 5]
```diff
@@ -15,20 +15,20 @@
...
 	sprite_sheets_refit = list(
```
Comment (Alberyk) [inline code/modules/clothing/spacesuits/void/void.dm]: This also uses bodytypes, not species's name.

File: code/game/machinery/suit_storage_unit.dm (hunk 2 above)
Comment (mikomyazaki) [reply code/game/machinery/suit_storage_unit.dm]: >_< Confusing when things that say 'species' mean 'bodytype'. Thanks.

**Comment by NonQueueingMatt** [comment]
Millions of conflicts, absent authors. Reopen when ready.

**Pull request closed without merging**